scala-xml/src/main/scala/scalaxml/ElemInstances.scala

```diff
@@ -44,7 +44,7 @@ trait ElemInstances {
     *
     * @return an XML element
     */
-  implicit def xml[F[_]](implicit F: Sync[F]): EntityDecoder[F, Elem] = {
+  implicit def xml[F[_]](implicit F: Async[F]): EntityDecoder[F, Elem] = {
```

Comment:
Does anything prevent this being `Concurrent`? That's the new constraint for most of the decoders.  Is it the `F.delay` on line 56?

Reply:
Exactly, I tried concurrent first but delay isn't available on concurrent

Reply:
I don't see much of a side effect here.  It consumes the `source`, but it's our source, from a strict byte array.  We don't need to demand a tighter constraint for this local mutation.  I think we could replace that `F.delay` with a `try / catch`, sort out the errors, and lose the `Async` constraint.

Reply:
thanks for the review, makes sense. looks also a bit simpler